openmetadata-ui/src/main/resources/ui/src/pages/DatabaseSchemaPage/DatabaseSchemaPage.component.tsx

```diff
@@ -93,6 +94,7 @@ const DatabaseSchemaPage: FunctionComponent = () => {
   const { t } = useTranslation();
   const { getEntityPermissionByFqn } = usePermissionProvider();
   const pagingInfo = usePaging(PAGE_SIZE);
+  const { storeCursor, getStoredCursor } = usePaging();
```

Comment:
We will be having both in line no. 96 no?
Also we are not passing default values from location?

Reply:
addressed